### DISCUSSION TITLE
Forced Resourcepacks

### DISCUSSION THREAD

**Discussion by KyllianGamer** [Minestom/Minestom#698] [Help] [answered]
When forcing a resource pack, but the player has their server resource packs set to "Disabled", is there a way to kick them after forcing didn't affect them (due to them having it on "Disabled") ?

**Comment by akeuben** [comment, marked as answer]
You could always listen to the [PlayerResourcePackStatusEvent](https://javadoc.minestom.net/net/minestom/server/event/player/PlayerResourcePackStatusEvent.html).

**Reply by KyllianGamer** [reply]
Hello, thank you for that. Didn't know that event existed. This event didn't get triggered in my case, though. But that was because of a different issue. By fixing that issue, my initial problem (of this thread) also got fixed. The issue was basically that I was sending the resource pack too soon, I guess, so the prompt never appeared because the player didn't load yet. I resolved this by scheduling it on the next tick.

**Reply by emortaldev** [reply]
You can mark the issue as resolved if this helped then :P
